src/test/java/com/cloudbees/jenkins/support/CheckFilterTest.java

```diff
@@ -0,0 +1,412 @@
+package com.cloudbees.jenkins.support;
```

Comment:
A big test to check a lot of components doing a bundle with the components to check and checking the content of the files inside.